src/main/webapp/app/exercises/programming/manage/update/programming-exercise-lifecycle.component.html

```diff
@@ -20,54 +21,65 @@
                 [startAt]="exercise.releaseDate"
                 [min]="exercise.releaseDate"
-                class="test-schedule-date"
+                class="test-schedule-date mr-2"
                 label="artemisApp.exercise.dueDate"
                 tooltipText="artemisApp.programmingExercise.timeline.dueDateTooltip"
             >
             </jhi-programming-exercise-test-schedule-date-picker>
-            <jhi-programming-exercise-test-schedule-date-picker
-                *ngIf="exercise.dueDate"
-                class="test-schedule-date"
-                [(ngModel)]="exercise.buildAndTestStudentSubmissionsAfterDueDate"
-                [startAt]="exercise.dueDate"
-                [min]="exercise.dueDate"
-                label="artemisApp.programmingExercise.timeline.afterDueDate"
-                tooltipText="artemisApp.programmingExercise.timeline.afterDueDateTooltip"
-            >
-            </jhi-programming-exercise-test-schedule-date-picker>
-            <div *ngIf="exercise.dueDate" class="test-schedule-date">
-                <div class="mb-2">
-                    <span class="font-weight-bold" jhiTranslate="artemisApp.programmingExercise.timeline.manualReview">
-                        Manual Review
-                    </span>
-                    <jhi-help-icon placement="top" text="artemisApp.programmingExercise.timeline.manualReviewTooltip"></jhi-help-icon>
-                </div>
-                <div
-                    *ngIf="!exercise.assessmentType || exercise.assessmentType === assessmentType.AUTOMATIC"
-                    class="btn btn-light scheduled-test scheduled-test--can-toggle btn-lifecycle btn-lifecycle--deactivated"
-                    (click)="toggleHasManualTests()"
-                >
-                    <fa-icon [icon]="'user-plus'" size="lg"></fa-icon>
-                </div>
-                <div
-                    *ngIf="exercise.assessmentType === assessmentType.SEMI_AUTOMATIC"
-                    (click)="toggleHasManualTests()"
-                    class="btn btn-light scheduled-test scheduled-test--can-toggle btn-lifecycle calendar-event-toggle"
-                >
-                    <fa-icon class="icon-static" [icon]="'user-check'" size="lg"></fa-icon>
-                    <fa-icon class="icon-remove" [icon]="'user-minus'" size="lg"></fa-icon>
-                </div>
-                <div
-                    *ngIf="!exercise.assessmentType || exercise.assessmentType === assessmentType.AUTOMATIC"
-                    class="text-danger"
-                    jhiTranslate="artemisApp.programmingExercise.timeline.notSet"
+            <div *ngIf="isExamMode || exercise.dueDate">
+                <jhi-programming-exercise-test-schedule-date-picker
+                    class="test-schedule-date mr-2"
+                    [(ngModel)]="exercise.buildAndTestStudentSubmissionsAfterDueDate"
+                    [startAt]="exercise.dueDate"
+                    [min]="exercise.dueDate"
```

Comment:
I'm not sure if that's a problem, but what would happen if `isExamMode` is `true` and `exercise.dueDate` is null?
Then this would also be `null` here.

Reply:
when I tried it out, it worked. I think startAt and min is ignored when the value is null